base/bucket.go

```diff
@@ -518,7 +517,7 @@ func GetBucket(spec BucketSpec, callback sgbucket.BucketNotifyFn) (bucket Bucket
 
 		if err != nil {
 			if pkgerrors.Cause(err) == gocb.ErrAuthError {
-				Warn("Unable to authenticate"+suffix+": %v", err)
+				WarnR("Unable to authenticate as user %q: %v", UD(username), err)
```

Comment:
While we're here, can the error be changed to a `%+v`?  Asking because the `pkgerror.Wrapf()` wrapped errors will dump a stack trace when `%+v` is used.

Reply:
I don't agree that we should be logging stack traces for Warnings.

Reply:
Yeah, I take it back for warnings, that will probably be too noisy.